becquerel/core/spectrum.py

```diff
@@ -529,6 +534,9 @@ def __mul__(self, other):
 
         return self._mul_div(other, div=False)
 
+    # This line will allow the right multiplication to work
+    __rmul__ = __mul__
```

Comment:
Shouldn't this go at the top of the class def?

Reply:
I forgot that in python order doesn't matter, I can move it to the top it probably is better to have it there

Reply:
Not sure why, it works locally, but travis doesn't like the `__rmul__` statement on top, so I moved it back down, I don't think it is worth the time figuring out why it wouldn't work on travis either.